packages/luxon/src/luxon-utils.ts

```diff
@@ -67,7 +67,10 @@ export default class LuxonUtils implements IUtils<DateTime> {
       return true;
     }
 
-    return this.date(value).equals(this.date(comparing));
+    const date = this.date(value);
+    if (!date) return false;
```

Comment:
It was more like a quick-fix @endOfWorkingDay kind of fix :D didn't check to carefully

As I get it the root problem is that `this.date(null)` on luxon doesn't return a luxon object so equals can't be called on undefined. This fix simply returns false in case this.date doesn't return a valid luxon date.

Tl;Dr;
It's fixing the production issue, but probably not the cleanest solution

Reply:
@dmtrKovalenko any update on this, anything i can do? This bug currently breaks our sign-up process